strings/base_error.h

```diff
@@ -179,7 +179,7 @@ namespace winrt::impl
 
     [[noreturn]] inline void __stdcall fallback_RoFailFastWithErrorContext(int32_t) noexcept
     {
-        std::terminate();
+        abort();
```

Comment:
Also at line 570 (`to_message`) of `base_error.h` and line 39 (`hex_to_uint`) of `base_types.h`? Otherwise, clang may complain "Not all code paths return a value."

Reply:
I didn't change those as Clang figured them out and didn't complain. We could switch everything over to `abort` or just wait for Clang to get with the standard.